### ISSUE TITLE
Alterar o minerador

### ISSUE THREAD

**Issue by fafato1** [HatsuZ/BitMiner-v3#1]
Olá, eu tenho estudado "bastante" sobre mineração e eu sei que o Minergate não é um bom minerador, então se precisar de ideia ou algum conhecimento sobre mineração eu posso ajudar, não sou bom com programação, mas venho estudando bastante.

**Comment by HatsuZ** [comment]
Opa, vc me tem adicionado no facebook ?

**Comment by fafato1** [comment]
Não, me adiciona lá https://www.facebook.com/profile.php?id=100008747957075
